bindings/aws/s3/s3.go

```diff
@@ -300,6 +301,10 @@ func (s *AWSS3) get(ctx context.Context, req *bindings.InvokeRequest) (*bindings
 		},
 	)
 	if err != nil {
+		var awsErr s3.Error
+		if errors.As(err, &awsErr) && *awsErr.Code == s3.ErrCodeNoSuchKey {
+			return nil, fmt.Errorf("object not found: %s", key)
```

Comment:
Let's not include the key in the result as it could contain PII (Personally Identifiable Information). Same for the other components

Reply:
Sure , Good catch !